src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -617,11 +614,6 @@ private void RunInstallCommands(InstallationContext installationContext, IEnumer
                     }
                     catch (Exception exception)
                     {
-                        if (exception.MustBeRethrown())
-                        {
-                            throw;
-                        }
-
```

Comment:
Yes, was also doubting on that. I prefer to have the minimal amount of exceptional code, but this isn't not backwardscompatible now. Thanks will fix this.